Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_app1.yaml

```diff
@@ -0,0 +1,80 @@
+
+id: f7abe9c1-1e6c-4317-b907-25769e7764c5
+name: Digital Shadows incident creation rule app 1
+kind: Scheduled
+properties:
+  displayName: Digital Shadows Incident rule - App 1
+  description: ''
+  severity: Medium
+  enabled: true
+  query: "let DSSearchLight_view  = view () { \n    DigitalShadows_CL\n    | where
+    app_s == \"app1\"\n    | extend \n                EventVendor=\"Digital Shadows\",\n
```

Comment:
the `app_s == \"app1\"` bit might need to change if you change the value for this field to `include` or `exclude` or something but I'll leave that up to you